packages/angular_devkit/build_angular/src/karma/schema.json

```diff
@@ -185,6 +185,10 @@
       "items": {
```

Comment:
Nit: can you add this also to https://github.com/angular/angular-cli/blob/master/packages/angular/cli/lib/config/schema.json?